src/js/elements/location_info_box.js

```diff
@@ -11,8 +12,10 @@ const metricTemplate = $('.location-highlight')[0].cloneNode(true);
 const infoContainer = $('.map__location-info', container);
 
 export class LocationInfoBox extends Observable {
-    constructor() {
+    constructor(_config) {
```

Comment:
I know you probably saw this all over the code but `_` variable names typically refer to private variables, which this is not. please change it to `config`

Reply:
👍